Tests/AnimationKeypathTests.swift

```diff
@@ -3,7 +3,7 @@
 
 import XCTest
 
-@testable import LottieCore
+@testable import Lottie
```

Comment:
This file accidentally wasn't included in the Xcode project, so wasn't being built / run on master